.github/workflows/release.yaml

```
@@ -47,6 +49,11 @@ jobs:
 
           script/release
 
+      - name: Upload release assets
+        uses: softprops/action-gh-release@v1
```

```suggestion
        uses: softprops/action-gh-release@v0.1.14
```
Let's use this so we do not get any supprises.
This is the latest version pushed to marketplace <https://github.com/marketplace/actions/gh-release>